goes16-l2-cmi.json

```diff
@@ -0,0 +1,569 @@
+{
+    "id": null,
+    "name": "GOES-16-L2-CMI",
+    "title": "GOES-16 Cloud & Moisture Imagery",
+    "description": "The GOES-16 Advanced Baseline Imager (ABI) L2 Cloud and Moisture Imagery provides 16 spectral bands with high temporal resolution over the American continent. The significance of the GOES-16 satellite for Brazil and South America lies in its location at longitude -75°, allowing it to offer comprehensive coverage of the continent and the oceanic regions of the Pacific and Atlantic. The ABI captures 2 visible, 4 near-infrared, and 10 infrared channels at resolutions ranging from 500m to 2km. This collection encompasses images acquired by the GOES-16 satellite (GOES-East) in full-disk mode, depicting nearly full coverage of the Western Hemisphere in a circular image. Important: note that other modes, such as CONUS and MESOSCALE, are not included in this collection. Cloud and Moisture Imagery product (CMIP) files are generated for each of the 16 ABI reflective and emissive bands. The collection captures CMIP product files into individual STAC Items for each observation from the GOES-16 satellite. It includes the original and full-resolution CMIP NetCDF files generated by INPE's GOES-R receive station. There is also a version for band 02, which originally has a resolution of 500m, degraded to 1km. Additionally, the collection contains CMIP files in a geographic regular grid format for each of the 16 spectral bands. These versions are generated by INPE using the process of mapping data from the original geostationary projection to a geographic regular grid (EPSG:4326). The geographical area used in the remapping process was chosen because it covers and highlights Brazil, South and Central America, with the following limits: lower left (lon: -116.0; lat: -56.0) and upper right (lon: -25.0, lat: 35.0). The image reprojection operation is performed using the gdalwarp utility (https://gdal.org/programs/gdalwarp.html) with the nearest neighbor resampling strategy defined. For more information, refer to the Beginner’s Guide to GOES-R Series Data (https://www.goesr.gov/downloads/resources/documents/Beginners_Guide_to_GOES-R_Series_Data.pdf), GOES-R Series Product Definition and Users Guide: Volume 5 (Level 2A+ Products) (https://www.goes-r.gov/products/docs/PUG-L2+-vol5.pdf) and the ABI Bands Quick Information Guides (https://www.goes-r.gov/mission/ABI-bands-quick-info.html).",
+    "temporal_composition_schema": null,
+    "composition_function": null,
```

Comment:
Is this dataset available following a Grid Reference system?
You should provide this information. If it isn't you can use `"grid_ref_sys": null,`